<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtcPropertiesClass</class>
 <widget class="QWidget" name="CtcPropertiesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CtcProperties</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>CTC</string>
   </property>
   <widget class="QComboBox" name="typeCombo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LCS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CTCI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CTCT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>QETH</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceNumber">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mainGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>431</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <widget class="QLineEdit" name="guestIP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000.000.000.000; </string>
    </property>
   </widget>
   <widget class="QLabel" name="guestIPLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Guest IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostIPLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hostIP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000.000.000.000; </string>
    </property>
   </widget>
   <widget class="QLabel" name="maskLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mask</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mask">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000.000.000.000; </string>
    </property>
   </widget>
   <widget class="QLabel" name="macLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mac">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>hh:hh:hh:hh:hh:hh; </string>
    </property>
   </widget>
   <widget class="QLabel" name="macLabel2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(optional)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tun">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>301</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tunLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tunnel Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="oatFilenameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OAT File Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="oatBrowseButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>21</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostPortLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host Port</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="hostPortSpinbox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1024</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>30880</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="lportSpinbox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1024</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>30880</number>
    </property>
   </widget>
   <widget class="QLabel" name="lportLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="buffsizeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Buffer Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="buffsizeSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>256</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>1500</number>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="tun0CheckBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>tun0 device</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
